.github/workflows/generated-linux-binary-conda-nightly.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       DOCKER_IMAGE: pytorch/conda-builder:cpu-main
       DESIRED_PYTHON: "3.8"
       build_name: conda-py3_8-cpu
+      use_split_build: false
```

Comment:
Can we make this variable be false by default so that we don't need to pass it ?